packages/tech-admin/package.json

```diff
@@ -18,6 +18,7 @@
   "devDependencies": {
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
+    "eslint": "^7.24.0",
```

Comment:
Do we need eslint in each package?  I think we can configure it on top-level. Please check